src/views/Player.vue

```diff
@@ -94,7 +101,8 @@ export default {
       journey: [],
       hasVideoPlayed: -1,
       sessionId: 1,
-      hasPlyrLoaded: false
+      hasPlyrLoaded: false,
+      retention: []
```

Comment:
Apparently a retention array was not defined as a data property, still was working fine though, no idea how.
Also there's a bug that sometimes when the response is uploaded on S3, there seems to be an extra `null` appended to the array. (Note: no where in the code am I calling `.push()` onto this array)
I'm hoping this would fix the bug.
I have not been able to replicate the bug